scripts/amoebagame.js

```diff
@@ -1,12 +1,12 @@
 /*
  * amoebagame.js
  */
-module.exports = function() {
+module.exports = function() {                              
     var id;
-    var name = "Evolution Game";
-    var species = ['Solosis', 'Lilligant', 'Krokorok', 'Minccino', 'Mankey', 'Arceus'];
-    var messages = ['solosis solosis', 'lilli-lilligant', 'krokokokokorok!', 'minccino?', 'ma-mankey mankey!', '-'];
-    var battles = {};
+                       var name = Evolution Game";            

```

Comment:
Missing semicolon.<br>Unclosed string.<br>Expected an assignment or function call and instead saw an expression.<br>'Evolution' is not defined.<br>Too many errors. (9% scanned).